src/gp-angular.js

```diff
@@ -597,6 +598,23 @@ module.directive('gpTranslate', ['$log', 'GlobalizationPipelineService', functio
                     if(DEBUG) {
                         $log.debug("[gp directive] promise resolved...value["+string+"]");
                     }
+
+                    // Check if the translation string has a variable expression
+                    var interpolateFn = $interpolate(string, true);

```

Comment:
Can this fail? I wonder if we should check `scope.values` first (that way one could use `values: false` to disable interpolation).  Again, not sure of the full syntax.